src/Visual.cc

```diff
@@ -193,6 +201,13 @@ Errors Visual::Load(ElementPtr _sdf)
   Errors geomErr = this->dataPtr->geom.Load(_sdf->GetElement("geometry"));
   errors.insert(errors.end(), geomErr.begin(), geomErr.end());
 
+  // Load the lidar reflective intensity if it is given
+  if (_sdf->HasElement("laser_retro"))
+  {
+    this->SetHasLaserRetro(true);
```

Comment:
This is being set inside `SetLasetRetro` too, do we need it in both places?

Reply:
You're right